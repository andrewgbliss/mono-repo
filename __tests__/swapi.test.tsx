import { render } from '@testing-library/react';
import Swapi from '../pages/swapi';
import fetchMock from 'jest-fetch-mock';

beforeEach(() => {
  fetchMock.doMock();
});

it('renders Swapi unchanged', () => {
  const { container } = render(<Swapi />);
  expect(container).toMatchSnapshot();
});
