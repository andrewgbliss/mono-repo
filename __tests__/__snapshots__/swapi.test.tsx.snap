// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Swapi unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-2 items-center font-mono"
  >
    <div
      class="h-32"
    />
    <h1
      class="text-2xl"
    >
      Swapi People
    </h1>
    <h5>
      Page: 
      1
    </h5>
    <div
      class="shadow-xl"
    >
      <table
        class="tbl"
      >
        <thead
          class="tbl-head"
        >
          <tr>
            <th
              class="tbl-cell w-64"
            >
              Name
            </th>
            <th
              class="tbl-cell w-32"
            >
              Height
            </th>
            <th
              class="tbl-cell w-32"
            >
              Birthyear
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
      <div
        class="border"
      />
      <div
        class="flex justify-between p-2"
      >
        <button
          class="btn btn-primary"
          disabled=""
        >
          Prev
        </button>
        <div>
          Loading...
        </div>
        <button
          class="btn btn-primary"
          disabled=""
        >
          Next
        </button>
      </div>
    </div>
  </div>
</div>
`;
